### PR TITLE
[Workers] Clarify log size limits

### PR DESCRIPTION
**Pull request by irvinebroque**
refs https://github.com/cloudflare/workerd/pull/910

### CODE REVIEW

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -336,3 +336,10 @@ Durable Objects have been built such that the number of Objects in the system do
 ## Image Resizing with Workers 
 
 When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on the applied limits.
+
+---
+
+## Log size
```
Comment (irvinebroque) [inline content/workers/platform/limits.md:342]: @deadlypants1973 — feel free to move around if you see a better home for this content on this page.

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -336,3 +336,10 @@ Durable Objects have been built such that the number of Objects in the system do
...
+
+- You can log (`console.log()`) a maximum of 128KB of data to the console for a single request. After you exceed this limit, further log statements will not be recorded or appear when tailing the logs of your Worker.
+- Refer to the [Workers Trace Event Logpush docs](/workers/observability/logpush/#limits) for information on the maximum size of fields sent to logpush destinations.
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
- You can log (`console.log()`) a maximum of 128 KB of data to the console for a single request. After you exceed this limit, further log statements will not be recorded or appear when tailing the logs of your Worker.
- Refer to the [Workers Trace Event Logpush documentation](/workers/observability/logpush/#limits) for information on the maximum size of fields sent to logpush destinations.
```

File: content/workers/platform/limits.md (hunk 1 above)
Comment (deadlypants1973) [reply content/workers/platform/limits.md:342]: This whole page needs a bit of an audit (some dupes and formating stuff) so I will take care of it then :)

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Review by kflansburg — APPROVED** [review]

File: content/workers/platform/limits.md [hunk 3]
```diff
@@ -336,3 +336,10 @@ Durable Objects have been built such that the number of Objects in the system do
 ## Image Resizing with Workers 
 
 When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on the applied limits.
+
+---
+
+## Log size
+
+- You can log (`console.log()`) a maximum of 128 KB of data to the console for a single request. After you exceed this limit, further log statements will not be recorded or appear when tailing the logs of your Worker.
```
Comment (irvinebroque) [inline content/workers/platform/limits.md]:
```suggestion
- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, reqeust metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).
```

File: content/workers/platform/limits.md [hunk 4]
```diff
@@ -336,3 +336,10 @@ Durable Objects have been built such that the number of Objects in the system do
 ## Image Resizing with Workers 
 
 When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on the applied limits.
+
+---
+
+## Log size
+
+- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, reqeust metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).
```
Comment (KianNH) [inline content/workers/platform/limits.md]:
```suggestion
- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, request metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).
```

File: content/workers/platform/limits.md [hunk 5]
```diff
@@ -336,3 +336,10 @@ Durable Objects have been built such that the number of Objects in the system do
 ## Image Resizing with Workers 
 
 When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on the applied limits.
+
+---
+
+## Log size
+
+- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, request metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](workers/observability/tail-workers/).
```
Comment (WalshyDev) [inline content/workers/platform/limits.md]:
```suggestion
- You can emit a maximum of 128 KB of data (across `console.log()` statements, exceptions, request metadata and headers) to the console for a single request. After you exceed this limit, further context associated with the request will not be recorded in logs, appear when tailing logs of your Worker, or within a [Tail Worker](/workers/observability/tail-workers/).
```

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**